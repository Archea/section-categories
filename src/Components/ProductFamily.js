import React, { Component } from 'react'

class ProductFamily extends Component {
  render() {
    return (
      <div>
        <div>
          ({this.props.productFamily.familyId}){' '}
          {this.props.productFamily.delimiter} {this.props.productFamily.name}
        </div>
      </div>
    )
  }

  _pickProductFamily = async () => {
    // ... you'll implement this
  }
}

export default ProductFamily
