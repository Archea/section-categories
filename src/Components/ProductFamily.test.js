import React from 'react'
import ReactDOM from 'react-dom'
import ProductFamily from './ProductFamily'
import renderer from 'react-test-renderer'

describe('ProductFamily', () => {
  const mocks = {
    productFamily: {
      id: '1',
      name: 'bar',
      delimiter: '-I-',
      familyId: '3',
    },
  }

  test('it at renders and matches snapshot', () => {
    const comp = renderer.create(<ProductFamily {...mocks} />)
    expect(comp).toBeDefined()
    expect(comp.toJSON()).toMatchSnapshot()
  })
})
