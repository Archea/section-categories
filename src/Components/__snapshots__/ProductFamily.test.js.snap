// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductFamily it at renders and matches snapshot 1`] = `
<div>
  <div>
    (
    3
    )
     
    -I-
     
    bar
  </div>
</div>
`;
